src/app/containers/CpsTable/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useContext } from 'react';
+import styled from '@emotion/styled';
+import { arrayOf, shape, string, oneOf } from 'prop-types';
+
+import { GEL_BODY_COPY } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { textBlockPropTypes } from '#models/propTypes/text';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import Blocks from '../Blocks';
+import CpsTableRow from './CpsTableRow';
+
+const StyledTable = styled.table`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_BODY_COPY}
+  border-spacing: 0;
+  width: 100%;
+`;
+
+const componentsToRender = {
+  tableHeaderRow: CpsTableRow,
+  tableRow: CpsTableRow,
+};
+
+const CpsTable = ({ blocks, supportedServices }) => {
+  const { service } = useContext(ServiceContext);
```

Comment:
The reason I put CpsTableRow and CpsTableCell inside the CpsTable folder was because they'll only ever be used as children of a CpsTable component. I'm very new to Simorgh though so maybe this was isn't conventional! I'm happy to move them to components if that's recommended?

Reply:
The makes total sense @paulcrane-bbc , we're working this stuff out a bit for ourselves at minute see: https://github.com/bbc/simorgh/pull/9166#discussion_r654403937 

I'd leave this as you have done it, it makes perfect sense given we done have a clear definition of components vs containers.